yarn.lock

```diff
@@ -1064,6 +1069,16 @@ ajv@*, ajv@^6.1.0, ajv@^6.10.0, ajv@^6.10.2, ajv@^6.12.2, ajv@^6.12.3:
     json-schema-traverse "^0.4.1"
     uri-js "^4.2.2"
 
+ajv@^6.1.0:
+  version "6.12.5"
```

Comment:
same here

Reply:
seems to have been a bug in yarn upgrade <pacake> for ever https://github.com/yarnpkg/yarn/issues/3967

Reply:
yarn-deduplicate solves it :)